### DISCUSSION TITLE
NUCLEAR ANIMATION MEME .

### DISCUSSION THREAD

**Discussion by trinhpxl** [keiwando/toonsquid#158] [Show and tell]
yeah hi i kinda caused object show community and animation meme community to blow up toonsquid xD 
heres an animation i did w toonsquid :D featuring my two main characters ! trinh (persona) and vinx (oc)

https://user-images.githubusercontent.com/115050959/218816307-de0131a0-2ef4-4637-9ade-8cb338531d5d.mp4

https://user-images.githubusercontent.com/115050959/218816352-570463b3-8180-4a9e-96d1-58146457fbef.mp4

**Comment by goldstargloww** [comment]
ayy another person who does animation memes helo :D

**Reply by trinhpxl** [reply]
IM LATE AS HELL BUT WSG

**Reply by goldstargloww** [reply]
iam also late hello again HJSFKG

**Comment by DinoAwakening** [comment]
AAAAAAGHH!! LIGHT THEME HUD!! D:

Nice animation btw ^^
